assets/js/modules/tagmanager/hooks/useExistingTagEffect.test.js

```diff
@@ -99,4 +99,153 @@ describe( 'useExistingTagEffect', () => {
 			true
 		);
 	} );
+
+	it( 'does not set change the useSnippet value when there is already a container ID on page load (container ID is same as existing tag)', async () => {
+		const account = factories.accountBuilder();
+		// eslint-disable-next-line sitekit/acronym-case
+		const accountID = account.accountId;
+		const containers = factories.buildContainers(
+			2,
+			// eslint-disable-next-line sitekit/acronym-case
+			{ accountId: account.accountId, usageContext: [ CONTEXT_WEB ] }
```

Comment:
`sitekit/acronym-case` screws us here. `accountId`. `accountID`, `account.accountId` all are present in this file with multiple cases of `eslint-ignore`-ing the error. Maybe we can enable the ignore file wide with an explanation and tidy up these assignments.

Reply:
Apologies for not spotting the difference in case there, and thanks for pointing this out!

Reply:
Cool. We do have [precedence](https://github.com/google/site-kit-wp/blob/52dc809d41a8f33f9c8d42218b4c3f68f538a2e0/assets/js/util/ip-cidr.js#L19-L24) of setting file-wide eslint ignore with an explanation to avoid repeating the ignore in every line. However, I will leave this one as is for now. Cheers.